homework4-2.py

```diff
@@ -0,0 +1,21 @@
+import copy
+# Use a list comprehension to construct the list
+# ['ab', 'ac', 'ad', 'bb', 'bc', 'bd']
+lst = [chr(97 + i) + chr(98 + j) for i in range(2) for j in range(3)]
+print(lst)
+# Use a slice on the above list to construct the list ['ab', 'ad', 'bc']
+lst1 = lst[::2]
+print(lst1)
+# Use a list comprehension to construct the list ['1a', '2a', '3a', '4a']
+lst = [str(el + 1) + 'a' for el in range(4)]
+print(lst)
+# Simultaneously remove the element '2a' from the above list and print it.
+lst.remove('2a')
```

Comment:
it need to be done in one line (simultaneously). We haven't looked through this functionality but we took it later on today's lection